src/test/unittest/ut_pmem2_config.c

```diff
@@ -46,3 +46,21 @@ ut_pmem2_config_delete(const char *file, int line, const char *func,
 
 	UT_ASSERTeq(*cfg, NULL);
 }
+
+/*
+ * prepare_config -- fill pmem2_config in minimal scope
+ */
+void
+ut_prepare_config(const char *file, int line, const char *func, 
+	struct pmem2_config **cfg, struct pmem2_source **src, int fd,
+	enum pmem2_granularity granularity)
+{
+	ut_pmem2_config_new(file, line, func, cfg);
+
+	if (fd != -1) {
```

Comment:
hmm, this makes src redundant in this path.
This should probably be somehow changed (i don't know how atm).

Reply:
Done.